GOVERNANCE.md

```diff
@@ -2,10 +2,10 @@
 
 This document outlines the governance model for Astro. This includes detailed descriptions of different roles, nomination processes, code review processes, and Code of Conduct enforcement.
 
-👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**  
+👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**
```

Comment:
The double spaces at the end of lines here were meaningful (they get rendered as a line break) and shouldn’t be removed:
```suggestion
👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**  
```

Reply:
I am going to add a line a line break instead, otherwise formatters go nuts